Docker build fails with npm ERR! enoent undefined ls-remote -h -t

After cloning the repo & doing a docker build, the build fails

```
docker build -t the_guard_b .
Sending build context to Docker daemon    874kB
Step 1/5 : FROM node:alpine
 ---> b7dc3fe8d4f8
Step 2/5 : COPY . /app
 ---> Using cache
 ---> b3078786a169
Step 3/5 : WORKDIR /app
 ---> Using cache
 ---> 73337e6e1c82
Step 4/5 : RUN npm install
 ---> Running in 418556c56db3
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/core-js-967e32e1/client/core.min.js.map'
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/eslint-8193e18d/LICENSE'
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/eslint-8193e18d/README.md'
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/regexpp-ba5eb1fa/index.js.map'
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/js-yaml-30f5b5c9/dist/js-yaml.min.js'
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/nedb-4d2e9171/browser-version/out/nedb.min.js'
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/acorn-1a822918/dist/acorn.mjs.map'
npm WARN tar ENOENT: no such file or directory, lstat '/app/node_modules/.staging/uri-js-0b5caf07/dist/esnext/schemes'
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/iconv-lite-96cd07ef/encodings/tables/gb18030-ranges.json'
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/iconv-lite-96cd07ef/encodings/tables/gbk-added.json'
npm WARN tar ENOENT: no such file or directory, open '/app/node_modules/.staging/iconv-lite-96cd07ef/encodings/tables/shiftjis.json'
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -2
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@gist.github.com/f9e184b78bbfc4419bc1ee70c238ca6f.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```

This is because alpine docker image doesn't contain `git` & needs to be added via

```
RUN apk add git
```

to the Dockerfile. 

I'll open a PR soon for this

--- a/Dockerfile
+++ b/Dockerfile
@@ -3,7 +3,7 @@ FROM node:alpine
 COPY . /app
 WORKDIR /app
 
-RUN npm install
+RUN apk add git && npm install
 
 CMD npm start
 

--- a/README.md
+++ b/README.md
@@ -27,7 +27,7 @@ You need to have [docker](https://docs.docker.com/engine/installation/linux/dock
 2. Clone this repository or [download zip](https://github.com/TheDevs-Network/the-guard-bot/archive/master.zip).
 3. Copy `example.config.js` to `config.js` and edit it.
 4. Run `docker build -t the_guard_bot .` to build image.
-5. Run `docker run --rm -itd the_guard_bot` to start the bot.
+5. Run `docker run -v $(pwd)/data:/app/data --rm -itd the_guard_bot` to start the bot.
 
 Now you can add the bot as **administrator** to your groups.
 

